### ISSUE TITLE
app stuck in upgrading status

### ISSUE THREAD

**Issue by lxm** [kubesphere/kubesphere#2951]
I hava a problem about app store,  i created a app with helm template, first upload a version 0.1.1 and then deploy it. then create a version 0.1.2 , when apply the update to the deployed app, openpitrix-hyperpitrix got a oom. the app stuck in upgrading status. is there any way to force change the status ?

![image](https://user-images.githubusercontent.com/1918195/92082595-58c17580-edf7-11ea-817b-b5608ab97154.png)

**Comment by zheng1** [comment]
```console
root@node1:~# kubectl exec -it -n kubesphere-system mysql-67cd66d5d-k2mvz -- mysql -uroot -ppassword
mysql> select * from cluster limit 1;
mysql> update cluster set transition_status='' where cluster_id='cl-xmzpm7yrwx9pkz';
Query OK, 0 rows affected (0.00 sec)
Rows matched: 0  Changed: 0  Warnings: 0
```

**Comment by lxm** [comment]
already do like that. may be a button should be added on UI

**Comment by zheng1** [comment]
Fixed in 3.1 nightly version
